index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Swiggy</title>
+      <link rel="stylesheet" href="styles/style.css">
+   </head>
+   <body>
+      <header class="header">
+         <img src="images/swiggyLogo.png" alt="Swiggy Logo">
+      </header>
+      <main>
+         <div class="banner">
+            <img src="images/momos.jpg" alt="Momos">
+            <h1>Punjabi Momos</h1>
+         </div>
+         <div class="content">
+            <div class="categories col-4">
+               <ul>
+                  <li class="recommended">Recommended</li>
```

Comment:
The category shown first might not always be recommended, rather could change to Favourites, Most ordered or any other category in future as per analytics

Should rename this class to `highlighted` as that's what this class is achieving